code/modules/mob/living/carbon/carbon.dm

```diff
@@ -500,17 +494,15 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
 						to_chat(src, "<span class='warning'>You can't crawl around in the ventilation ducts with items!</span>")
 						return
 
-			visible_message("<b>[src] scrambles into the ventilation ducts!</b>", "You climb into the ventilation system.")
-			src.loc = vent_found
-			add_ventcrawl(vent_found)
-
-	else
-		to_chat(src, "<span class='warning'>This ventilation duct is not connected to anything!</span>")
+			show_ventcrawl(entrance_found)
+			loc = entrance_found // This can not be a forcemove. It will break the ventcrawl UI
```

Comment:
Can you figure out *why* `forceMove()` breaks this? Force setting the `loc` var without other sanity checks is bad practice

Reply:
I can try, though this kinda falls into legacy code that I don't really understand why territory. I was just adding a comment for the sake of others trying to fix this in the future. It wasn't code I added.